app/src/main/java/com/framgia/forder/screen/shopmanagement/ItemShopInfoViewModel.java

```diff
@@ -1,20 +1,30 @@
 package com.framgia.forder.screen.shopmanagement;
 
 import android.content.Context;
+import android.databinding.BaseObservable;
+import android.databinding.ObservableInt;
 import com.framgia.forder.R;
+import com.framgia.forder.data.model.ShopDomain;
 import com.framgia.forder.data.model.ShopInfo;
+import com.framgia.forder.data.model.ShopManagement;
 
 /**
  * Created by levutantuan on 5/8/17.
  */
 
-public class ItemShopInfoViewModel {
+public class ItemShopInfoViewModel extends BaseObservable {
     private final Context mContext;
     private final ShopInfo mShopInfo;
+    private final ShopDomain mShopDomain;
+    public final ObservableInt mButtonJoinDomain;
```

Comment:
WARNING: Variable 'mButtonJoinDomain' must be private and have accessor methods.